addon/components/sortable-group.js

```diff
@@ -20,35 +27,55 @@ export default Component.extend({
     @type Ember.NativeArray
   */
   items: computed({
-    get() { return a(); }
-  }),
+    get() {
+      return a(); 
+    },
+  }), 
+
   /**
-    Vertical position for the first item.
+    @method _getFirstItemPosition
+    @private
+  */
+  _getFirstItemPosition: function(direction) {
+    let element = this.element;
+    let stooge = $('<span style="position: absolute" />');
+    let prependedStoogePosition = stooge.prependTo(element).position();
+    let result;
+
+    if (direction === 'x') {
+      result = prependedStoogePosition.left;
+    }
+    if (direction === 'y') {
+      result = prependedStoogePosition.top;
+    }
+
+    stooge.remove();
+
+    return result;
+  },
+
+  /**
+    Position for the first item.
 
     @property itemPosition
     @type Number
   */
   itemPosition: computed({
     get() {
-      let element = this.element;
-      let stooge = $('<span style="position: absolute" />');
-      let result = stooge.prependTo(element).position().top;
-
-      stooge.remove();
-
-      return result;
+      return this._getFirstItemPosition(this.get('direction'));
     }
-  }).volatile(),
+  }).volatile().readOnly(),
 
   /**
     @property sortedItems
     @type Array
   */
-  sortedItems: computed('items.@each.y', {
+
+  sortedItems: computed('items.@each.y', 'items.@each.x', {
```

Comment:
Technically this also depends upon `direction`, although we don’t expect that to ever change.